shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -229,4 +238,58 @@ public static String failMetricName(String metricName) {
             return metricName + "_fail";
         }
     }
+
+    /**
+     * Convenient method to concastenate two Strings using dot.
+     *
+     * @param element1 the first String object.
+     * @param element2 the second String object.
+     * @return String object with element1 and element2 connected with dot.
+     */
+    public static String name(String element1, String element2) {
```

Comment:
Some comments here: i) the method name `name` is confusing and the parameter names are ambiguous. As a method, you may want to describe the action that is happening inside, like `buildMetricName(String prefix, String metricName)` or something of the sort. ii) The description of the method looks as it was a general string concatenation, but this is using a dot between two strings for a reason: to respect the naming structure of metrics. So, maybe the description of the method should refer what is doing and why. iii) Typo: `concastenate` -> `concatenate`.

Reply:
fixed